pages/wishlist.html

```diff
@@ -0,0 +1,282 @@
+              id="searchbar"
+              placeholder="Search products..."
+            />
+          </div>
+          <button class="hamburger">
+            <i class="fas fa-bars"></i>
+          </button>
+          <nav class="nav-list">
+            <button class="nav-list-close-btn">
+              <i class="fas fa-2x fa-times"></i>
+            </button>
+            <ul class="nav-menu h-100">
+              <li class="nav-item">
+                <a href="../../index.html"
+                  ><button class="btn btn-md btn-outline px-md">
+                    Logout
+                  </button></a
+                >
+              </li>
+              <li class="nav-item">
+                <div class="badge">
+                  <div class="fas fa-heart badge-icon txt-md px-sm"></div>
+                  <div class="badge-count top-right flex-center badge-yellow">
+                    6
+                  </div>
+                  <a href="#" class="ecom-heart-icon"></a>
+                </div>
+              </li>
+              <li class="nav-item">
+                <div class="badge">
+                  <div
+                    class="fas fa-shopping-cart badge-icon txt-md px-sm"
+                  ></div>
+                  <div class="badge-count top-right flex-center badge-yellow">
+                    2
+                  </div>
+                  <a href="#" class="ecom-basket-icon"></a>
+                </div>
+              </li>
+            </ul>
+          </nav>
+        </header>
+        <div class="mob-search-input hide">
+          <div class="fas fa-search mob-search-icon"></div>
+          <label for="searchbar"></label>
+          <input
+            class="input-round input-md border-2"
+            type="search"
+            name="searchbar"
+            id="searchbar"
+            placeholder="Search products..."
+          />
+        </div>
+      </div>
+      <div class="main my-2">
+        <main class="max-width-1200 mx-auto">
+          <h2 class="page-title txt-bold mx-auto py-md">My Wishlist</h2>
+          <hr />
+          <section
+            class="page-content grid grid-cols-auto grid-col-1 grid-cols-2 gap-1 mx-auto px-lg"
+          >
+            <div class="card mx-auto">
+              <div class="card-img">
+                <img src="../assets/card-img1.png" alt="vertical card" />
+              </div>
+              <div class="card-title flex-col">
+                <p class="txt-md txt-bold mb-sm">MV33 Brimmed Cap</p>
+                <div class="flex-row align-center">
+                  <div class="dis-flex flex-wrap align-center">
+                    <p class="txt-md txt-bold mr-sm">₹3,099/-</p>
+                    <s class="strike-color txt-semibold">₹3,400/-</s>
+                  </div>
+                </div>
+              </div>
+              <hr />
+              <div class="card-btns">
+                <button class="btn btn-wish btn-md txt-bold txt-reg w-100">
```

Comment:
You can also add a button or a badge , so that user can remove the item from the wishlist.

Reply:
Great suggestion, not sure how I forgot about it!
Will add it in the next commit. Thanks for heads up!